src/atwho.js

```diff
@@ -101,7 +101,9 @@ define([
                     callbacks: {
                         matcher: function(flag, subtext) {
                             var match, regexp;
-                            regexp = new RegExp('(\\s+|^)' + RegExp.escape(flag) + '([\\w_/]*)$', 'gi');
+                            // Match text that starts with the flag and then looks like a path.
+                            // CKEditor reserves the right to insert arbitrary zero-width spaces, so watch for those.
```

Comment:
Yeah, happily it doesn't get into the XML - I didn't dig into exactly why not, but am guessing that we write out the editor's actual data (which doesn't have the zws) rather than looking at the HTML markup of the editor (which does).